### ISSUE TITLE
Mobile styles for invited users table

### ISSUE THREAD

**Issue by AileenCGN** [TryGhost/Ghost#8744]
### Issue Summary

![img_0875](https://user-images.githubusercontent.com/8037602/28508446-f4be3cca-704b-11e7-8c44-dad23c6cd85b.jpg)

### Technical details:

* Ghost Version: 1.0
* Browser/OS:  Chrome/iOS 10.3.3 iPhone 7 Plus

**Comment by farhd** [comment]
@AileenCGN I'd like to take a look at it

**Comment by farhd** [comment]
also:
![screenshot at jul 26 11-48-39](https://user-images.githubusercontent.com/6986862/28615230-b6334f6e-71f8-11e7-9780-b415e2db3bc3.png)
![screenshot at jul 26 11-49-39](https://user-images.githubusercontent.com/6986862/28615229-b63296fa-71f8-11e7-80aa-018d05f09b40.png)

'apps-card' & 'apps-configured' styles could be the issue. I'll take a look at it

**Comment by AileenCGN** [comment]
@farhd: Did you make any progress for this?

**Comment by farhd** [comment]
@AileenCGN yes, but I couldn't push my changes to my remote git, as my setup wasn't complete. I never worked with submodules before. I will take care of it today in the evening though.

**Comment by AileenCGN** [comment]
@farhd Cool! Maybe [this guide](https://docs.ghost.org/docs/working-with-the-admin-client) helps you with your development environment for Ghost-Admin?

**Comment by farhd** [comment]
https://github.com/TryGhost/Ghost-Admin/pull/813

**Comment by kevinansfield** [comment]
TryGhost/Ghost-Admin#813 has been merged, it's improved some areas but there's still room for further improvement. Here are screenshots at iphone6 sizes of the relevant areas (we aren't optimising for anything smaller than this):

<img width="249" alt="screen shot 2017-08-17 at 11 57 49" src="https://user-images.githubusercontent.com/415/29409436-19e50640-8344-11e7-8b28-88881905c8ce.png">
<img width="248" alt="screen shot 2017-08-17 at 11 58 14" src="https://user-images.githubusercontent.com/415/29409437-19e50794-8344-11e7-92c5-9b51ff22b82c.png">
<img width="249" alt="screen shot 2017-08-17 at 11 59 16" src="https://user-images.githubusercontent.com/415/29409438-1a054b80-8344-11e7-8f7f-3761fed2efee.png">

**Comment by nithindavid** [comment]
@kevinansfield Which all pages need responsive fixes?

**Comment by kevinansfield** [comment]
https://github.com/TryGhost/Ghost-Admin/pull/903 has very nearly resolved this issue but there are still a couple of edge cases remaining:

<img width="375" alt="screen shot 2017-11-09 at 12 23 04" src="https://user-images.githubusercontent.com/415/32605531-b0aa5426-c549-11e7-8a63-608d49f556c6.png">

<img width="376" alt="screen shot 2017-11-09 at 12 24 19" src="https://user-images.githubusercontent.com/415/32605542-b55b6b2c-c549-11e7-9dc8-1787735769f1.png">

**Comment by kevinansfield** [comment]
Oops, this wasn't supposed to be closed

**Comment by martgnz** [comment]
Thanks for merging @kevinansfield! Will work on the edge cases on a new PR 👍
